gadgets/trace_bind/README.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: trace_bind
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# trace_bind
+
+The trace_bind gadget is used to stream socket binding syscalls.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/trace_bind:latest [flags]
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/trace_bind:latest [flags]
+        ```
+    </TabItem>
+</Tabs>
+## Flags
+
+### `--ignore_errors`
+Show only events where the bind succeeded. By default all events are traced.
+
+### `--pid`
+Show only bind events generated by this particular PID. By default all events are traced.
+
+## Guide
+
+This example shows how to use this gadget.
```

Comment:
This text is almost always repeated directly as the first sentence under 'guide'.

Is this necessary?